### PR TITLE
Activated Knife Changes

### PR DESCRIPTION
**Pull request by Nozarati**
## About The Pull Request

updating weapons to fit the current knives available

## Details

<details>
  <summary>Expand</summary>
Changes the Butterfly Knife and Switchblade to moddable tools just like the other knives. Changes the crafting recipe for butterfly knife to be included in crafting menu, and use anything with screwdriving quality. Commented out previous way of crafting the knife to avoid confusion, and edited (hopefully) all instances of these knives appearing to be the new moddable tool version. Slightly changed material costs for switchblade to reflect its description(0.5 gold), and made them more rare in the spawns.

</details>

## Screenshots

<details>
  <summary>Expand</summary>
https://imgur.com/a/nChV0Ls 
</details>

### CODE REVIEW

File: code/game/objects/items/weapons/improvised_components.dm [hunk 1]
```diff
@@ -29,14 +29,14 @@
...
-		qdel(src)
-		return
+//obj/item/weapon/material/butterflyhandle/attackby(obj/item/W as obj, mob/user as mob)
+//	if(istype(W,/obj/item/weapon/material/butterflyblade))
+//		var/obj/item/weapon/material/butterflyblade/B = W
+//		to_chat(user, "You attach the two concealed blade parts.")
+//		new /obj/item/weapon/material/butterflyconstruction(user.loc, B.material.name)
+//		qdel(W)
+//		qdel(src)
+//		return
```
Comment (ACCount12) [inline code/game/objects/items/weapons/improvised_components.dm]: Don't comment out old dead code unless you plan on actually using it in the future, and maybe not even then. Remove it, along with the items and all.

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 2]
```diff
@@ -158,6 +158,74 @@
...
+	item_state = "[initial(item_state)]_on"
+	to_chat(user, SPAN_NOTICE("You press a button on the handle and [src] slides out."))
+	playsound(user, 'sound/weapons/flipblade.ogg', 15, 1)
+	icon_state = "switchblade_open"
+	edge = TRUE
+	sharp = TRUE
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	..()
+/obj/item/weapon/tool/knife/switchblade/turn_off(mob/user)
```
Comment (ACCount12) [inline code/game/objects/items/weapons/tools/simple_weapons.dm]: Keep formatting sane: leave at least one empty line between code blocks. This applies to every instance of this.

**Comment by Jackofashepard** [comment]
Looks good

**Comment by TorinoFermic** [comment]
As required by https://github.com/discordia-space/CEV-Eris/pull/4712

**Pull request closed without merging**